### PR TITLE
8328367: Convert java/awt/Component/UpdatingBootTime test to main

### PR DESCRIPTION
**Pull request by azuev-java**
Convert test to manual main
Move test to the Component folder

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8328367](https://bugs.openjdk.org/browse/JDK-8328367): Convert java/awt/Component/UpdatingBootTime test to main (**Bug** - P4)

### Reviewers
 * [Damon Nguyen](https://openjdk.org/census#dnguyen) (@DamonGuy - Committer)
 * [Prasanta Sadhukhan](https://openjdk.org/census#psadhukhan) (@prsadhuk - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/18355/head:pull/18355` \
`$ git checkout pull/18355`

Update a local copy of the PR: \
`$ git checkout pull/18355` \
`$ git pull https://git.openjdk.org/jdk.git pull/18355/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 18355`

View PR using the GUI difftool: \
`$ git pr show -t 18355`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/18355.diff">https://git.openjdk.org/jdk/pull/18355.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/18355#issuecomment-2004551489)

### CODE REVIEW

File: test/jdk/java/awt/Component/UpdatingBootTime.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 6461933 7194219
+ * @summary adjust system boot time in nowMillisUTC() frequently
```
Comment (prsadhuk) [inline test/jdk/java/awt/Component/UpdatingBootTime.java:27]: Guess you need @library for PassFailJFrame

File: test/jdk/java/awt/Component/UpdatingBootTime.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        frame.pack();
+        return frame;
+    }
+
+    public static void main(String[] args) throws InterruptedException,
+            InvocationTargetException {
+        String instructions =
+                """
+            1) In the test window press "Press me" button.
+            2) A two timestamps should be printed.
```
Comment (prsadhuk) [inline test/jdk/java/awt/Component/UpdatingBootTime.java:74]: grammar issue...guess it should be just "Two timestamps should..."

File: test/jdk/java/awt/Component/UpdatingBootTime.java (hunk 1 above)
Comment (azuev-java) [reply test/jdk/java/awt/Component/UpdatingBootTime.java:27]: Sure. Here and in all other tests.

**Review by DamonGuy — APPROVED** [review]
LGTM

**Review by prsadhuk — APPROVED** [review]

**Pull request closed without merging**